toplevel/coqtop.ml

```diff
@@ -257,8 +257,7 @@ type custom_toplevel =
   ; opts : Coqargs.t
   }
 
-
-let init_toploop opts =
+let init_stm opts =
```

Comment:
This is better named as `init_document`, the stm has its own initialization there.

Reply:
Done.